odc/stats/plugins/tcw_percentiles.py

```diff
@@ -22,48 +27,77 @@ class StatsTCWPC(StatsPluginInterface):
-        },
-        input_bands: Sequence[str] = [
-            "blue",
-            "green",
-            "red",
-            "nir",
-            "swir1",
-            "swir2",
-            "fmask",
-            "nbart_contiguity",
-        ],
-        output_bands: Sequence[str] = ["wet", "bright", "green"],
+        coefficients: Dict[str, Dict[str, float]] = None,
+        input_bands: Sequence[str] = None,
+        output_bands: Sequence[str] = None,
+        cloud_filters: Dict[str, Iterable[Tuple[str, int]]] = None,
         **kwargs,
     ):
-        super().__init__(input_bands=input_bands, **kwargs)
-        self.coefficients = coefficients
-        self.output_bands = output_bands
+
+        self.cloud_filters = cloud_filters if cloud_filters is not None else {}
+        if coefficients is None:
+            self.coefficients = dict(
+                [
+                    (
+                        "wet",
+                        dict(
+                            [
+                                ("blue", 0.0315),
+                                ("green", 0.2021),
+                                ("red", 0.3102),
+                                ("nir", 0.1594),
+                                ("swir1", -0.6806),
+                                ("swir2", -0.6109),
+                            ]
+                        ),
+                    ),
+                    (
+                        "bright",
+                        dict(
+                            [
+                                ("blue", 0.2043),
+                                ("green", 0.4158),
+                                ("red", 0.5524),
+                                ("nir", 0.5741),
+                                ("swir1", 0.3124),
+                                ("swir2", 0.2303),
+                            ]
+                        ),
+                    ),
+                    (
+                        "green",
+                        dict(
+                            [
+                                ("blue", -0.1603),
+                                ("green", -0.2819),
+                                ("red", -0.4934),
+                                ("nir", 0.7940),
+                                ("swir1", -0.0002),
+                                ("swir2", -0.1446),
+                            ]
+                        ),
+                    ),
+                ]
+            )
+        self.input_bands = (
+            [
+                "blue",
+                "green",
+                "red",
+                "nir",
+                "swir1",
+                "swir2",
+                "fmask",
+                "nbart_contiguity",
+            ]
+            if input_bands is None
+            else input_bands
+        )
+
+        super().__init__(input_bands=self.input_bands, **kwargs)
+        self.output_bands = (
+            ["wet", "bright", "green"] if output_bands is None else output_bands
+        )
 
```

Comment:
All of this formatting change was done to keep black happy.

Reply:
thanks!